### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by lisandroseia**
In this branch I made the styling for the webpage, kindly review, any optional suggestion is well received :smile:

### CODE REVIEW

**Comment by omar-labana** [comment]
@lisandroseia Nice dogo GIF.

File: src/components/addBook.js [hunk 1]
```diff
@@ -30,16 +36,18 @@ const AddBook = () => {
         submitBook(title, category);
       }}
     >
+      <h2>Add new book</h2>
```
Comment (omar-labana) [inline src/components/addBook.js:39]: - [ ] According to the design, there should be a styled `hr` element between the book form and the booklist. please add it.

File: src/components/renderBook.js [hunk 2]
```diff
@@ -1,21 +1,56 @@
...
+          <p className="completed">completed</p>
+        </div>
+      </div>
+      <div className="book-progress">
+        <p className="current-chapter">Current chapter</p>
+        <p className="chapter">introduction</p>
+        <button className="update-btn" type="button">
+          Update progress
+        </button>
+      </div>
```
Comment (omar-labana) [inline src/components/renderBook.js]: - [ ] According to the design, there should be a vertical separator between these two sections. please add it.

**Review by omar-labana — CHANGES_REQUESTED** [review]
Hi @lisandroseia, 

Excellent work so far! 👌
You have implemented almost all the required features. ✔️ 
There are some cases that you need to adjust but you are almost done! 😀

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_N/A_

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/renderBook.js (hunk 2 above)
Comment (lisandroseia) [reply src/components/renderBook.js]: Thanks for the review! I implemented the changes

**Review by Emmyn5600 — APPROVED** [review]
Hi @lisandroseia 

STATUS: APPROVED :partying_face: 

Wooow👏, Your project is complete!  There is nothing else to say other than... it's time to merge it. 👏

Congratulations! 🎉 

Optional suggestions
 
- N/A
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**